build-parent/pom.xml

```diff
@@ -495,6 +487,26 @@
         <formatter.goal>validate</formatter.goal>
         <impsort.goal>check</impsort.goal>
       </properties>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>net.revelc.code.formatter</groupId>
+            <artifactId>formatter-maven-plugin</artifactId>
+          </plugin>
+          <plugin>
+            <groupId>net.revelc.code</groupId>
+            <artifactId>impsort-maven-plugin</artifactId>
+          </plugin>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-enforcer-plugin</artifactId>
+          </plugin>
+          <plugin>
+            <groupId>org.jacoco</groupId>
+            <artifactId>jacoco-maven-plugin</artifactId>
+          </plugin>
+        </plugins>
+      </build>
```

Comment:
By default those plugins refactor the code according to the codestyles. Can you move them back to the default profiles so that we get our codebase formatted when we compile the code?

Reply:
done